_docs-2/getting-started/quickstart.md

```diff
@@ -137,15 +137,19 @@ Each method above has instructions below.
 
 Start Accumulo processes (tserver, master, monitor, etc) using command below:
 
+    accumulo master
     accumulo tserver
+    accumulo monitor
 
 The process will run in the foreground. Use ctrl-c to quit.
 
 ### Run Accumulo services
 
 Start Accumulo services (tserver, master, monitor, etc) using command below:
 
+    accumulo-service master start
     accumulo-service tserver start
+    accumulo-service monitor start
```

Comment:
You can use the accumulo-cluster cmd to start all of the services either on a single node or on multiple nodes - these commands are only necessary if you elect to run each manually (or say you need to restart just one)  Maybe it would be more user-friendly if the accumulo-cluster command section came before this section?

Also, for a complete installation I believe that you might be missing necessary services (gc). Then maybe this could be
`accumulo [service-name] or accumulo-service [service-name] start | stop` with a more complete list of the required service names?

Reply:
@EdColeman i'm a newbie with accumulo and while trying to create a table i simply didn't know why it hangs.
I've spent more than an hour debugging and pondering about it, and afterwards I came to see the fact that there are more services to start, not only the `tserver`.

Obviously my modification is quite naive because there are probably more services. So what you are suggesting with `accumulo [service-name]` fits much better.

The quick start for me wasn't so quick :)

I came across the sample code https://github.com/apache/accumulo-examples/blob/main/docs/sample.md which gives a few pointers on how to work with the accumulo shell - I find it quite good for a quick start - to get a feeling about what accumulo actually does.

Reply:
The dos start with consider using fluo-uno  - that's the easiest on ramp that I know of. Is there a reason that you did not try it? Would chaning / adding wording there have made you more likely to use fluo-uno?

Moving the cluster command before the individual commands might have made your experience easier - with accumulo-cluster start, things should have been started and removes the beginner from needing five commands instead of one.

But with that, uno fetch accumulo, uno start accumulo is way easier than setting up accumulo from scratch.

Reply:
A bit unrelated, but another small issue I've encountered was the `ClassNotFoundError` regarding zookeeper's KeeperException class.

Cause of it : 
`accumulo-env.sh`
```
 
 
 
 CLASSPATH="${CLASSPATH}:${lib}/*:${HADOOP_CONF_DIR}:${ZOOKEEPER_HOME}/*:${HADOOP_HOME}/share/hadoop/client/*"
```

replaced it with 

```
 
 
 
 CLASSPATH="${CLASSPATH}:${lib}/*:${HADOOP_CONF_DIR}:${ZOOKEEPER_HOME}/lib/*:${HADOOP_HOME}/share/hadoop/client/*"
```

(added `lib` after ZOOKEEPER_HOME)

Reply:
> But with that, uno fetch accumulo, uno start accumulo is way easier than setting up accumulo from scratch.

I'm looking now through the code of `uno` and see that it downloads everything what is needed. I am sorry, but due to the fact that I have already zookeeper and hadoop on my machine I thought to opt to simply install the bin archive of accumulo.

Reply:
I tried with both apache zookeeper 3.5.9 and zookeeper 3.7.0
In both of them, the .jar libraries are located under `ZOOKEEPER_HOME/lib` directory.

Reply:
What version of Accumulo?

Reply:
Since that is unrelated, let's not bog this issue down with that issue. See apache/accumulo#1530 for more.